docs/builtins-reference.md

```diff
@@ -127,12 +123,12 @@ kmeans(x, centers, iter.max, nstart)
 
 ### Arguments
 
-| Name   | Type            | Default  | Description |
-| :----  | :-------------  | -------- | :------------------------------- |
-| x      | Matrix[Numeric] | required |The input Matrix to do KMeans on..|
-|centers | Int             | -------- |The no. of cenetrs                |
-|iter.max|Int              |10        |Max no. of iterations allowed     |
-|nstart  |Int              |10        |No. of random starting positions  |
+| Name     | Type            | Default  | Description |
+| :--      | :--             | --       | :--         |
```

Comment:
Can you revert changes of this line. I was only talking about `centers` corresponding!

Reply:
Done. Only the changes to 'centers' were done this time.